2-mandatory/4-stocks.js

```diff
@@ -34,61 +34,80 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  let totalAvg = [];
+  for (let eachStock of closingPricesForAllStocks) {
+    let avg = 0;
+    let total = 0;
+    for (let price of eachStock) {
+      total += price;
+      avg = total / eachStock.length;
+      avg = Math.round(avg * 100) / 100;
```

Comment:
1. You code works. However, the calculation at line 43 - 44 shall be run once only for each stock. The extra calculations of avg before the last calculation were being overwritten in each loop.  
2. Better use separate variables to store the average value and the rounded up average